apps/ledger-live-mobile/e2e/helpers.ts

```diff
@@ -65,31 +68,39 @@ export async function clearTextByElement(elem: Detox.NativeElement) {
   return elem.clearText();
```

Comment:
Don't we want to use await, here?
https://wix.github.io/Detox/docs/next/api/actions/#cleartext

Reply:
Fixed